vars/createOpenshiftDockerSecret.groovy

```diff
@@ -0,0 +1,19 @@
+Closure call(String username, String password, String email) {
```

Comment:
Typing the params here will cause an exception to be thrown if someone tries to use it with params of a different type (not sure if there's an exception if it can coerce the type of the input to match)